.gitignore

```diff
@@ -75,3 +75,5 @@ node_modules/
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+
+.idea/
```

Comment:
InteliJ user 👀 

Can  we add a comment above on what this is ignoring?

Reply:
I'd recommend adding this to your global gitignore.

(Also an IntelliJ user here 😎)